### PR TITLE
Meeting listings improvements

### PR DESCRIPTION
**Pull request by phgross**
- Disable selection links for listings without checkbox column.
- Extend proposal listing columns, with the new columns (state, comittee, proposed action)
- Added tests for the Proposal and the SumittedProposalListing

Additionally I made the `proposed_action` field also available in the proposal add- and edit-form

@deiferni please have a look ...

### CODE REVIEW

File: opengever/meeting/tabs/submittedproposallisting.py [hunk 1]
```diff
@@ -24,16 +24,17 @@ def get_base_query(self):
...
-        {'column': 'initial_position',
-         'column_title': _(u'column_initial_position',
-                           default=u'Initial Position')},
-        )
+        columns = super(SubmittedProposalListingTab, self).columns
+        for col in columns:
+            if col.get('column') == 'title':
+                col['transform'] = proposal_title_link
 
+        return columns
```
Comment (deiferni) [inline opengever/meeting/tabs/submittedproposallisting.py]: missing newline :wink:

**Comment by deiferni** [comment]
looks :+1: 
could you rebase onto `master` and add translations for the new table headings for proposal listings (`column_...`). I had to add these translations manually to this file https://github.com/4teamwork/opengever.core/blob/master/opengever/tabbedview/locales/opengever.tabbedview-manual.pot in `opengever.tabbedview`.

**Comment by phgross** [comment]
@deiferni I've rebased the PR and update the meeting translations: 

I've also cleanup some bad or not equal default values of translations, to avoid all the warnings wich are displayed when using `bin/i18n-build`

**Comment by deiferni** [comment]
nice, thank you!

**Pull request merged**